src/config/dynamic-theme-fixes.config

```diff
@@ -6206,17 +6206,61 @@ gog.com
 
 INVERT
 i.icn.icn--close
+svg.icon-wrapper-icon
+svg.big-spot__add-to-cart-icon
+svg.ic-svg:not(.button__icon):not(.productcard-slider__nav-icon)
+.productcard-thumbnails-slider-pagination
+.carousel-pagination__page
+.big-spot__carousel-pages-container
+.big-spots__arrow-icon
+.discover-games-more__icon
 
 CSS
 .product-tile__info:hover {
     background-color: var(--darkreader-neutral-background) !important;
 }
+a.product-tile__content {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.explore-galaxy {
+    border-top: 0 !important;
+}
```

Comment:
Hmm interesting that dark reader is modifying the border-top, I would like to fix this in the code. You can remove it for now.